apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/MonitoredResourceUtils.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
I think that would be a good idea, not point in duplicating the code.

My only worry is that (IIRC) MonitoredResource isn't a logging specific thing.  I think it's general for many Google APIs.